test/translixir_happy_path_test.exs

```diff
@@ -0,0 +1,276 @@
+         status_code: 200
+       }}
+    end)
+
+    result =
+      Translixir.entity_tx!(
+        client_pid,
+        :jorge,
+        ~U[2020-11-04 00:07:29.057Z],
+        ~U[2020-11-04 00:07:29.057Z]
+      )
+
+    assert result ==
+             %{
+               "crux.db/content-hash": %Eden.Tag{
+                 name: "crux/id",
+                 value: "d5c474af4ced822d951d0d2da2d75cf946bca62c"
+               },
+               "crux.db/id": %Eden.Tag{
+                 name: "crux/id",
+                 value: "33f927344e079e00d3fa45d8833b04e735223eec"
+               },
+               "crux.db/valid-time": ~U[2020-11-04 00:07:29.057Z],
+               "crux.tx/tx-id": 2,
+               "crux.tx/tx-time": ~U[2020-11-04 00:07:29.057Z]
+             }
+  end
+
+  test "entity history" do
+    Translixir.MockHTTPoison
+    |> expect(:get, fn
+      "http://localhost:3000/entity-history/33f927344e079e00d3fa45d8833b04e735223eec?sort-order=asc&with-docs=true",
+      [{"Content-Type", "application/edn"}] ->
+        {:ok,
+         %HTTPoison.Response{
+           body:
+             "({:crux.tx/tx-time #inst \"2020-11-01T02:33:13.371-00:00\", :crux.tx/tx-id 0, :crux.db/valid-time #inst \"2020-11-01T02:33:13.371-00:00\", :crux.db/content-hash #crux/id \"d5c474af4ced822d951d0d2da2d75cf946bca62c\", :crux.db/doc {:crux.db/id :jorge, :name \"hello\"}}
+            {:crux.tx/tx-time #inst \"2020-11-01T02:33:47.525-00:00\", :crux.tx/tx-id 1, :crux.db/valid-time #inst \"2020-11-01T02:33:47.525-00:00\", :crux.db/content-hash #crux/id \"d5c474af4ced822d951d0d2da2d75cf946bca62c\", :crux.db/doc {:crux.db/id :jorge, :name \"hello\"}}
+            {:crux.tx/tx-time #inst \"2020-11-04T00:07:29.057-00:00\", :crux.tx/tx-id 2, :crux.db/valid-time #inst \"2020-11-04T00:07:29.057-00:00\", :crux.db/content-hash #crux/id \"d5c474af4ced822d951d0d2da2d75cf946bca62c\", :crux.db/doc {:crux.db/id :jorge, :name \"hello\"}})",
+           status_code: 200
+         }}
+    end)
+
+    result =
+      Translixir.entity_history(
+        Client.new("localhost", "3000"),
+        "33f927344e079e00d3fa45d8833b04e735223eec",
+        :asc,
+        true
+      )
+
+    assert result ==
+             [
+               %{
+                 "crux.db/content-hash": %Eden.Tag{
+                   name: "crux/id",
+                   value: "d5c474af4ced822d951d0d2da2d75cf946bca62c"
+                 },
+                 "crux.db/doc": %{"crux.db/id": :jorge, name: "hello"},
+                 "crux.db/valid-time": ~U[2020-11-01 02:33:13.371Z],
+                 "crux.tx/tx-id": 0,
+                 "crux.tx/tx-time": ~U[2020-11-01 02:33:13.371Z]
+               },
+               %{
+                 "crux.db/content-hash": %Eden.Tag{
+                   name: "crux/id",
+                   value: "d5c474af4ced822d951d0d2da2d75cf946bca62c"
+                 },
+                 "crux.db/doc": %{"crux.db/id": :jorge, name: "hello"},
```

Comment:
I saw that sometimes has `%Eden.Tag` and other times no. Is there a reason to use Eden.Tag?

Reply:
This depends on the result of `Eden.decode`, also sometimes for testing reasons I just encode it back